### PR TITLE
Minor display fixes

### PR DESCRIPTION
**Pull request by tracer4b**
Fixes some typos in lang files and formula displays.
Also fixes sodium nitrate being Na(NO₃)₃ (:trollface:)

### CODE REVIEW

File: src/main/java/gregicality/science/api/unification/materials/GCYSFirstDegreeMaterials.java [hunk 1]
```diff
@@ -130,8 +130,9 @@ public static void init() {
                 .fluid()
                 .color(0xFFFFC8)
                 .flags(DISABLE_DECOMPOSITION)
-                .components(Hydrogen, 2, Platinum, 1, Palladium, 1, Chlorine, 6)
-                .build();
+                .components(Hydrogen, 4, Platinum, 1, Palladium, 1, Chlorine, 12)
+                .build()
+                .setFormula("H2(Pt,Pd)Cl6", true);
```
Comment (TechLord22) [inline src/main/java/gregicality/science/api/unification/materials/GCYSFirstDegreeMaterials.java:135]: By CEu convention commas are not to be included in formulae. Please revert this change.

**Review by TechLord22 — CHANGES_REQUESTED** [review]

File: src/main/java/gregicality/science/api/unification/materials/GCYSFirstDegreeMaterials.java (hunk 1 above)
Comment (tracer4b) [reply src/main/java/gregicality/science/api/unification/materials/GCYSFirstDegreeMaterials.java]: I've reverted it for now, but this is something I'd like to discuss

**Review by TechLord22 — APPROVED** [review]
